cmd/cosign/cli/generate/generate_key_pair.go

```diff
@@ -85,7 +86,12 @@ func GenerateKeyPairCmd(ctx context.Context, kmsVal string, args []string) error
 		return err
 	}
 
-	if cosign.FileExists("cosign.key") {
+	fileExists, err := icos.FileExists("cosign.key")
+	if err != nil {
+		return errors.Wrap(err, "error checking if cosign.key exists")
```

Comment:
please use `fmt.Errorf` instead of `errors.Wrap`  the `github.com/pkg/errors` is archived and we should use the go native one.